Packages/mormot_base.lpk

```diff
@@ -16,8 +16,10 @@
         <UnitOutputDirectory Value="lib\$(TargetCPU)-$(TargetOS)"/>
       </SearchPaths>
       <Other>
-        <OtherDefines Count="1">
-          <Define0 Value="USEZEOS"/>
+        <CustomOptions Value="-dHASPACKAGES"/>
+        <OtherDefines Count="2">
+          <Define0 Value="HASPACKAGES"/>
```

Comment:
@synopse I've chose `HASPACKAGES` because it already exists a `USEPACKAGES` (disabled by default) in `Synopse.inc`